src/dashboard/src/components/administration/views.py

```diff
@@ -223,6 +227,127 @@ def storage(request):
     system_directory_description = 'Available storage'
     return render(request, 'administration/locations.html', locals())
 
+def usage(request):
+    usage_dirs = _usage_dirs()
+
+    return render(request, 'administration/usage.html', locals())
+
+def _usage_dirs(calculate_usage = True):
+    # Get shared directory path
+    clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
+    config = ConfigParser.SafeConfigParser()
+    config.read(clientConfigFilePath)
+
+    shared_dir = config.get('MCPClient', 'sharedDirectoryMounted')
+
+    # Get DIP uploads directory
+    dip_upload_dir = os.path.join(shared_dir, 'watchedDirectories/uploadDIP')
```

Comment:
No, I meant that having a literal `/` in a parameter to os.path.join kinda defeats the purpose, since os.path.join behaves differently depending on the OS and `/` pins it to a particular OS.  Should be ```os.path.join(shared_dir, 'watchedDirectories', 'uploadedDIPs')```.  Here I'm also assuming shared_dir is constructed with os.path.join (or read from a config file with an OS-correct value).